sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -246,13 +246,24 @@ class ExpressionParserSuite extends PlanTest {
     // Basic window testing.
     assertEqual("foo(*) over w1", UnresolvedWindowExpression(func, WindowSpecReference("w1")))
     assertEqual("foo(*) over ()", windowed())
+    assertEqual("foo(*) over (partition by a)", windowed(Seq('a)))
```

Comment:
You shall also cover the `null` cases, such as `foo(*) over (partition by null)`